math/min/bitwisemin.go

```diff
@@ -0,0 +1,16 @@
+// bitwisemin.go
+// description: Gives min of two integers
+// details:
+// implementation of finding the minimum of two numbers using only binary operations without using conditions
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see bitwisemin_test.go
+
+package min
+
+func Bitwise(base int, values ...int) int {
```

Comment:
Apologies for asking this 😅 I know this was my suggestion but its incorrect and I just noticed that this would panic if no values are provided. Could you change the signature to this instead?
```suggestion
func Bitwise(base int, value int, values ...int) int {
```

Reply:
Done! =)